### DISCUSSION TITLE
Welcome to pdpipe Discussions!

### DISCUSSION THREAD

**Discussion by shaypal5** [pdpipe/pdpipe#78] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other people using or developing `pdpipe`. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Also, remember the documentation site - with plenty of tutorials and introductory materials - can be found here:
https://pdpipe.readthedocs.io/en/latest/

**Comment by carbonleakage** [comment]
Hello there!

I accidentally came across this repo and have been having quite a bit of fun working with it! Congrats @shaypal5 on the excellent package architecture! I am quite eager to also contribute to further enhancing the package, so was wondering if there are any specific issues where I can be of help? I checked through the issues marked as "good first issue" but looks like someone is working is on most of these.

Just as a background, I have been working with python for more than 5 years and would consider myself an intermediate/advanced pandas user. I have done some basic work with open source packages but would love to make a deeper dive :)

**Comment by shaypal5** [comment]
Hey! Yay!

This is so exciting! :)

Hmmm. I have a couple of ideas.

Something that comes to mind right now is more advanced handles that allow for easy and natural stage creation.

See the new fly handles, `drop_rows_where` and `keep_rows_where`:

https://pdpipe.readthedocs.io/en/latest/starting/fly/

Also, I don't think anything is really being actively worked on. I think most of those promises were forgotten by the people who wanted to help, naturally.

Anyway, I'll surely come up with something. Or maybe you will? If you've played around with the package you've probably found some missing pieces or things that take to long to express.

I'll write again. Just want to welcome you. :)

**Comment by carbonleakage** [comment]
Thanks, I'll explore further!

**Reply by shaypal5** [reply]
I tagged you on a couple of issues that might interest you, @carbonleakage .
